v2/cf-nodejs-logging-support/src/test/config.test.js

```diff
@@ -0,0 +1,126 @@
+const expect = require('chai').expect;
+const importFresh = require('import-fresh');
+const customConfig = require('./config-test.json');
+
+describe('Test configuration', function () {
+
+    console.log(customConfig);
+
+    var result;
+
+    beforeEach(function () {
+        log = importFresh("../../build/main/index");
```

Comment:
These tests are focusing on the `Config` class but import the `RootLogger` instance. Can you convert it to a unit test for the `Config` class? We should not rely on the `RootLogger` class forwarding methods for unit tests.

Do not get me wrong, we should definitely have some acceptance tests for the new API methods. But these would not test if the config gets parsed correctly, but if the log output is correct for a given configuration. I think this can be done in a separate PR later on.

Reply:
Makes sense. I created new subfolders to separate acceptance and unit test. In the unit test folder I added a new unit test file for the `Config` class which will be completed in a separate PR, so far it has only a base structure with the methods to be tested.